### DISCUSSION TITLE
Custom Boot logo for Swiss

### DISCUSSION THREAD

**Discussion by ScytheandGenesisGames** [emukidid/swiss-gc#741] [General]
Does Swiss have an option to use this which is a custom bootloader for gamecube - https://github.com/OffBroadway/flippyboot-ipl/ the boot logo does work via from start up but not when selecting a game, I would love to have my gamecube to show a platinum colored Screen when it's booting the game.

**Comment by Extrems** [comment]
It's not custom, but you can enable boot through IPL to get the logo.

**Reply by ScytheandGenesisGames** [reply]
How I can do that so it does show my custom boot with the platinum colored cube?

**Reply by Extrems** [reply]
You can't.
